src/stores/utilities/run.js

```diff
@@ -141,6 +141,10 @@ const createRunType = ({ id }) => {
  * creates a JSONAPI payload for a run
  */
 const createPayload = ({ id, run, plates, wells, smrtLinkVersion, instrumentType }) => {
+  // Removes read only attributes
```

Comment:
Advisory: I think it would be better to a separate method for this e.g. removeReadOnlyAttributes.

I guess we could have a list of read only attributes somewhere.

deleting stuff from objects in-flight can have side effects.